test/helpers.py

```diff
@@ -113,8 +113,14 @@ def reset_akamai_purges():
     requests.post("http://localhost:6789/debug/reset-purges")
 
 def verify_akamai_purge():
-    response = requests.get("http://localhost:6789/debug/get-purges")
-    purgeData = response.json()
-    if len(purgeData["V3"]) is not 1:
-        raise Exception("Unexpected number of Akamai v3 purges")
+    timeout = time.time() + 5
```

Comment:
Technically this is a `deadline`. :-)

Reply:
Tbh I just copy/pasted this from `fetch_until`, which I believe you originally wrote 😉.

Reply:
Lol, love to pick nits in my own old code. :-)